Formula/r.rb

```diff
@@ -0,0 +1,172 @@
+class RDownloadStrategy < SubversionDownloadStrategy
+  def stage
+    cp_r File.join(cached_location, "."), Dir.pwd
+  end
+end
+
+class R < Formula
+  desc "Software environment for statistical computing"
+  homepage "https://www.r-project.org/"
+  url "https://cran.rstudio.com/src/base/R-3/R-3.4.1.tar.gz"
+  sha256 "02b1135d15ea969a3582caeb95594a05e830a6debcdb5b85ed2d5836a6a3fc78"
+  head "https://svn.r-project.org/R/trunk", :using => RDownloadStrategy
```

Comment:
Yeh, tempting to just remove it and figure out a better approach that doesn't involve a custom download strategy.